### ISSUE TITLE
VIC does not return image name

### ISSUE THREAD

**Issue by lazarin** [vmware/vic#2089]
**VIC Version:**

0.4.0 / build #4031

**Code**

1. Create alpine container
2. Inspect the container CLI / RestAPI
3. Image element should return image name, but it has internal VIC's image id

```ts
$ docker run -i -t -d --name cont1 alpine sh -c 'echo string'
$ docker inspect cont1
```

**Expected behavior (as it is on regular docker host):**
```
...
"Config": {
    ...
    "Image": "alpine",
    ...
},
```
**Actual behavior:**
```
...
"Config": {
    ...
    "Image": "ad84c222f6a3d862d40da1ab38291737800571718bb70b1804b957d0dfcbc036",
    ...
},
```

**Comment by kjplatz** [comment]
Verified this on current Master build (commit 89092b8ccd02e8b97473c2aeea7528bc71e08936):

Running against Docker v1.11.2:
* ```docker run -i -t -d --name cont1 alpine sh -c 'echo string'```
* ```docker inspect cont1 | grep Image```
```
    "Image" : "sha256:4e38e38c8ce0b8d9041a9c4fefe786631d1416225e13b0bfe8cfa2321aec4bba",
    "Config" : {
        "Image": "alpine",
    }
```

Running against VIC:
```
        "Image": "ad84c222f6a3d862d40da1ab38291737800571718bb70b1804b957d0dfcbc036",
        "Config" : {
            "Image": "ad84c222f6a3d862d40da1ab38291737800571718bb70b1804b957d0dfcbc036",
        }
```

**Comment by corrieb** [comment]
This would explain why <insert portal name here> displays the image ID rather than the image name when you inspect a container. This is a highly visible UX issue in <insert portal name here>, so if it's a simple fix, it would be great to target this at 0.5.5
